alerter.c

```diff
@@ -12,8 +16,19 @@ int networkAlertStub(float celcius) {
 }
 
 void alertInCelcius(float farenheit) {
+    
+    int returnCode;
+    
     float celcius = (farenheit - 32) * 5 / 9;
-    int returnCode = networkAlertStub(celcius);
+    
+    if(Current_Env == TESTCODE_ENV)

```

Comment:
Using conditional code to execute test or production code increases complexity as the code size grows. Maintenance becomes difficult.
Test code and production code should be maintained separately.

Production code should not be aware of test environment or production envt.

Use dependency injection to solve this problem.